src/modules/auth/Auth.router.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
```

Comment:
`import * as Nesto from 'nesto';` je jako losa praksa, jer [tree shaking](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking) onda ne radi... Sto znaci da importujes i ono sta ti treba i ono sta ti ne treba i onda na kraju bude veca aplikacija i duze se ucitava... 
Tako da je uvek bolje da se koristi `import { nestoStoKoristis } from 'nesto'`. 

Takodje zanimljiva extenzija na ovu temu u VSCode je [Import Cost](https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost); 

P.S. Vidim da je Tosa to radio jos na nekim mestima... Tako da ako mozes, popravi i to...